nautobot/extras/tests/test_filters.py

```diff
@@ -1401,10 +1402,17 @@ def test_color(self):
 
     def test_search(self):
         params = {"q": "active"}
-        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 1)
-        value = self.queryset.values_list("pk", flat=True)[0]
+        q = Q(name__icontains="active") | Q(slug__icontains="active") | Q(content_types__model__icontains="active")
```

Comment:
I assume the content-types filter is just matching the `search()` implementation? We don't actually have any models at present that match this, right?

Reply:
It shouldn't match on anything but we can't say for sure anymore now that the test fixtures are being randomly generated by factories. This should be the same Q object that's generated by the search filter's `generate_query` method but I didn't want to leverage a method in the filterset to test the filterset.